doc/classes/Rect2i.xml

```diff
@@ -70,41 +82,51 @@
 			<return type="Rect2i" />
 			<param index="0" name="to" type="Vector2i" />
 			<description>
-				Returns a copy of this [Rect2i] expanded so that the borders align with the given point.
+				Returns a copy of this rectangle expanded to include the given [param to] point, if necessary.
```

Comment:
I am pretty sure this change is wrong. It was specifically modified because, when expanded to the right or bottom, the `to` point is not included in the rectangle by the `has_point`. (and this is on purpose)

Reply:
https://github.com/godotengine/godot/pull/80217